prow/plugins/config_test.go

```diff
@@ -216,16 +216,22 @@ func TestSetApproveDefaults(t *testing.T) {
 
 		commandHelpLink         string
 		expectedCommandHelpLink string
+		prProcessLink           string
+		expectedPrProcessLink   string
 	}{
 		{
 			name:                    "set commandHelpLink default",
 			commandHelpLink:         "",
+			prProcessLink:           "",
 			expectedCommandHelpLink: "https://go.k8s.io/bot-commands",
+			expectedPrProcessLink:   "https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process",
 		},
 		{
 			name:                    "keep commandHelpLink value",
 			commandHelpLink:         "https://prow.k8s.io/command-help",
+			prProcessLink:           "https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#the-code-review-process",
```

Comment:
mind changing the `prProcessLink` here? This test would pass even if the functionality of setting it was broken as the input value is the same as the default

Reply:
Sure, I figured the default being `https://git.k8s.io/community` was different enough from `https://github.com/kubernetes/community` but I made it more explicitly different than the default now.